### PR TITLE
Admin collaboration

### PR DESCRIPTION
**Pull request by StevenMalaihollo**
Closes https://github.com/bcc-code/bcc-membership/issues/2901

### CODE REVIEW

File: server/src/permissions/appAbility.ts [hunk 1]
```diff
@@ -103,9 +94,18 @@ const rolePermissions: Record<VoteRoleName, DefinePermissions> = {
             visibility: PollResultVisibility['Non Public'],
         });
         can('find', 'user', {churchID: {$in: votingAdminFor}});
+
+        can(['get', 'find'], 'polling-event', {
+            'participantFilter.org': {$in: votingAdminFor},
+        });
+        can(['find', 'get'], 'polling-event', {creatorId: user.personID});
+
         can('get', 'polling-event-report', {
```
Comment (JakubC-projects) [inline server/src/permissions/appAbility.ts:103]: isn't this permission obsolete now?

File: server/src/permissions/appAbility.ts [hunk 2]
```diff
@@ -76,16 +72,11 @@ const rolePermissions: Record<VoteRoleName, DefinePermissions> = {
...
-        });
+        can(['find', 'get'], 'polling-event');
     },
     VotingAdmin(user, {can}) {
         const votingAdminFor = user.roles.filter((r: UserRole) => r.enumName === 'VotingAdmin')[0].orgIDs;
-        can('create', 'poll');
-        can('update', 'poll');
-        can('remove', 'poll');
-        can('patch', 'poll');
+        can('manage', 'poll');
```
Comment (JakubC-projects) [inline server/src/permissions/appAbility.ts:79]: This looks like an excessive permission

**Comment by JakubC-projects** [comment]
I think we should consider having the admin panel update in real time, because it may be used by multiple people at the same time

**Review by JakubC-projects — APPROVED** [review]

File: server/src/permissions/appAbility.ts (hunk 2 above)
Comment (StevenMalaihollo) [reply server/src/permissions/appAbility.ts:79]: No this is exactly the same as it was. Before the VotingAdmins could allready do all the actions

File: server/src/permissions/appAbility.ts (hunk 1 above)
Comment (StevenMalaihollo) [reply server/src/permissions/appAbility.ts:103]: Right, true

**Comment by StevenMalaihollo** [comment]
> I think we should consider having the admin panel update in real time, because it may be used by multiple people at the same time

Agreed, will do so in the next PR.

**Pull request merged**